docker/configs/config.yaml

```diff
@@ -0,0 +1,76 @@
+extensions:
```

Comment:
might be nice to name this file "collector-config.yaml" for extra clarity

Reply:
yeah, that was the plan, but appears the Docker otelcol looks for config.yaml by default -- I can get around this by adding a `command` to the docker-compose, but though that might raise questions...